README.md

```diff
@@ -13,7 +13,8 @@ designed to work well with [Bazel](http://bazel.io).
 git_repository(
     name = "subpar",
     remote = "https://github.com/google/subpar",
-    commit = "HEAD",
+    tag = "1.0",
+    sha256 = "Fill in sha256 here",
```

Comment:
... and having submitted this without testing, of course 'sha256' is documented but doesn't actually work.